A24Group/ssGWT-lib
# Feature staffing shift component 3296

Sorry for the big commit.

Added a staffing shift component that can be used to select a start and end date.
Also added test cases for the above component.

A24group/Triage#3296

Plz review

Test
- Check out
- Run test cases
- Compile jar
- Copy jar over to triage
- Run cap a24deploy:gwt:application_compile
- Make sure everything run as expected.
